codebuild_specs/run_cdk_canary_e2e_tests.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.2
+env:
+  shell: bash
+  variables:
+    AMPLIFY_DIR: /root/.npm-global/lib/node_modules/@aws-amplify/cli-internal/bin
+    AMPLIFY_PATH: /root/.npm-global/lib/node_modules/@aws-amplify/cli-internal/bin/amplify
+    CI: true
+    CODEBUILD: true
+    NODE_OPTIONS: --max-old-space-size=8096
+phases:
+  build:
+    commands:
+      - codebuild-breakpoint
+      - source ./shared-scripts.sh && _setupE2ETestsLinux
+      - source ./shared-scripts.sh && _runE2ETestsLinux
```

Comment:
Shouldn't this be "_runCanaryTest" instead of "_runE2ETestsLinux"? Maybe I'm misunderstanding the intent of this?

Reply:
Good catch, this should be `_runCDKTestsLinux` which we use for handling retries, credential setting etc for the CDK construct based E2Es.